attributes/default.rb

```diff
@@ -453,3 +453,6 @@
 default["hopsworks"]['check_nodemanager_status']              = "false"
 
 default['hopsworks']['azure-ca-cert']['download-url']         = "https://cacerts.digicert.com/DigiCertGlobalRootG2.crt"
+
+#livy
+default['livy_startup_timeout']           = "240"
```

Comment:
```suggestion
default['hopsworks']['livy_startup_timeout']           = "240"
```

Reply:
The dml uses ['hopsworks']['livy_startup_timeout'] so I assume you missed to add the hopsworks part in the attribute